R/library.R

```diff
@@ -257,3 +257,44 @@ measure_interaction <- function(intxn) {
     composite(intxn$name, colnames(intxn$data),weights = mode_A)
   }
 }
+
+kurt <- function(x, na.rm = FALSE) {
```

Comment:
Few questions:
1. did you write these functions yourself or did you get them from somewhere?
2. do we need to handle all forms of objects (vector, matrix, data.frame etc.) or just vectors?

Reply:
1. I altered them from elsewhere. 
2. Nope, so I have altered the code to just use apply

Reply:
ok, in future just run by me once where we are taking code from.  we should know what license applies to that code.